app/(admin)/admin/notifications/page.tsx

```diff
@@ -38,33 +48,111 @@ const AdminNotifications = (): JSX.Element => {
    */
   const handleSubmit = async (event: React.FormEvent): Promise<void> => {
     event.preventDefault();
-    await sendEmailNotifications({ content, sendEmailUsers, subject });
+    await sendEmailNotifications({
+      content,
+      sendEmailUsers: groupUsers,
+      subject,
+    });
   };
 
+  /**
+   * Function to handle radio selection logic.
+   * @param value - Value of the radio buttons.
+   */
+  const handleRadioChange = (value: string): void => {
+    setEmailRecipients(value);
```

Comment:
> If we are doing it this way, then I think we should do `value: string[]` and then `setGroupUsers(value)` so that way every time we click the radio button, it already has the userId's set for emailing.

@alexappleget I was going to change, this, however, with the addition of the `setEmailSubjects` state I mentioned in my [comment above](https://github.com/LetsGetTechnical/gridiron-survivor/pull/490#discussion_r1794410881), I will keep it as is, as the `setEmailSubjects` does not expect an array.